Operational-Procedures/Scheduler/night-time-operations/index.rst

```diff
@@ -0,0 +1,255 @@
+.. _scheduler-night-time-operation:
+
+##############################
+Scheduler Night-time Operation
+##############################
+
+.. _scheduler-night-time-operation-initial-optical-alignement:
+
+Initial Optical Alignement
+==========================
+
+When the Scheduler is initialized at the beginning of the night, the first thing it will perform is an optical alignement, by running a curvature wavefront sensing script.
```

Comment:
alignment (remove e). In the case of the Auxiliary Telescope, this means running the curvature wavefront sensing script.